### PR TITLE
Cursor improvements

### PR DESCRIPTION
**Pull request by hvennekate**
Some more improvements after I discussed the cursor with the actual users ;)

### CODE REVIEW

File: doc/manual/en/ch11_configuration.tex [hunk 1]
```diff
@@ -663,6 +663,10 @@ \section{Look / Screen Layout}\label{sec:interface-appearance}
   {\bf Box}: Draws boxes around each InfoBox. \\
   {\bf Tab}: Draws a tab at the top of the InfoBox across the 
   title. \\
+\item[Cursor size*]  Control the size of the cursor. Default is 2.
```
Comment (hvennekate) [inline doc/manual/en/ch11_configuration.tex]: Question: do we document features that depend on the build target?

File: po/ca.po [hunk 2]
```diff
@@ -5722,7 +5722,7 @@ msgstr ""
 
 #: src/Dialogs/Settings/Panels/AirspaceConfigPanel.cpp:69
 #: src/Dialogs/Settings/Panels/InterfaceConfigPanel.cpp:169
-#: src/Dialogs/Settings/Panels/LayoutConfigPanel.cpp:60
```
Comment (hvennekate) [inline po/ca.po]: Forgot to update those the last time around...

File: src/Screen/TopWindow.hpp [hunk 3]
```diff
@@ -144,6 +144,7 @@ class TopWindow : public ContainerWindow {
 #endif
 #ifdef DRAW_MOUSE_CURSOR
   uint8_t cursor_size = 1;
```
Comment (hvennekate) [inline src/Screen/TopWindow.hpp]: This should really be "2"

File: src/Screen/FB/TopWindow.cpp [hunk 4]
```diff
@@ -91,17 +91,23 @@ TopWindow::OnPaint(Canvas &canvas)
...
 
   const BulkPixelPoint p[] = {
     { m.x, m.y },
     { m.x + shortDistance, m.y + shortDistance },
     { m.x, m.y + longDistance },
   };
 
-  canvas.SelectBlackPen();
-  canvas.SelectWhiteBrush();
+  unsigned penSize = cursor_size / 2 + cursor_size % 2;
```
Comment (hvennekate) [inline src/Screen/FB/TopWindow.cpp]: The poor man's version of rounding up...

**Comment by MaxKellermann** [comment]
Updates to po/* should not be part of a PR, because this can break easily and cause conflicts.
New Canvas methods should be a separate commit.
Cursor size change and inverted colors should be separate commits. It is hard to review a commit when you do two different things mixed in one commit.

**Comment by hvennekate** [comment]
> Updates to po/* should not be part of a PR, because this can break easily and cause conflicts.
> New Canvas methods should be a separate commit.
> Cursor size change and inverted colors should be separate commits. It is hard to review a commit when you do two different things mixed in one commit.

HI Max! thanks for the review -- will adapt accordingly.
Just one question: where do I put the po/* file changes? This is basically to update the line numbers and add the translations for the cursor stuff.

**Comment by hvennekate** [comment]
Changes have been split into multiple commits and po/* file changes removed.

File: doc/manual/en/ch11_configuration.tex (hunk 1 above)
Comment (MaxKellermann) [reply doc/manual/en/ch11_configuration.tex]: Yes, please - maybe with a footnote?

**Comment by MaxKellermann** [comment]
> Just one question: where do I put the po/* file changes?

Not at all. Let me update those.

**Comment by hvennekate** [comment]
> > Just one question: where do I put the po/* file changes?
> 
> Not at all. Let me update those.

:joy:  If only I had known that sooner! You sure you do Chinese as well?

**Pull request merged**